server-spi-private/src/main/java/org/keycloak/models/utils/EnabledFeaturesSupplier.java

```diff
@@ -0,0 +1,18 @@
+package org.keycloak.models.utils;
```

Comment:
Please remove. This information is available on server info (serverInfo.profileInfo.)